test-node.bash

```diff
@@ -292,6 +301,7 @@ if $force_init; then
     echo == Funding validator and sequencer
     docker-compose run scripts send-l1 --ethamount 1000 --to validator --wait
     docker-compose run scripts send-l1 --ethamount 1000 --to sequencer --wait
+    docker-compose run scripts send-l1 --ethamount 1000 --to espresso-sequencer --wait
```

Comment:
What is the purpose of this espresso-sequencer account? Does it correspond to the hotshot commitment contract?

Reply:
Yes. It needs to commit/deploy the hotshot contract. There are some other ways to do this, like funding in the Genesis Block. But I think it is not a good idea to change so many files and this special account may be useful in later testing.